src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Automatic anomaly detection 
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how automatic anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+Automatic anomaly detection is the easiest way for you and your team to detect and analyze unusual behavior in your system. It does this using the following methods:
+
+1. Anomaly detection monitors metric data reported by an APM agent, building a model of your typical application dynamics, and focuses on three key golden signals: throughput, response time, and errors.
```

Comment:
This is expanding to other agents, not just APM. Maybe just remove `APM`?

Reply:
The third golden signal is considered `Error Rate` but errors might be easier to read/understand

Reply:
done! Thanks.